app/models/payable_group.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class PayableGroup < ApplicationRecord
+  include PaperTrailTraceable
+
+  has_many :invoices
+  has_many :payments, as: :payable
+
+  def customer
+    invoices.first&.customer
```

Comment:
since we have reference to a customer in the table, do we still need it?

Reply:
good spot 👍